src/providers/BackgroundBlurProvider/docs/BackgroundBlurProvider.stories.mdx

```diff
@@ -0,0 +1,109 @@
+Background Blur related logs can be found in the browser developer tools when the `BackgroundBlurProvider` is used within the app component tree.
+
+This provider is independent from `MeetingProvider`. You can put `BackgroundBlurProvider` wherever you want in the component tree. The `BackgroundBlurProvider` loads the required worker assets
+when mounted, which may impact performance.
+
+## State
+
+```typescript
+{
+  // Whether Background Blur is supported. The default value is undefined, then changes to true or false.
+  isBackgroundBlurSupported: boolean | undefined;
+
+  // A function to transform a video input device to a DefaultVideoTransformDevice.
+  createBackgroundBlurDevice: (device: Device) => Promise<DefaultVideoTransformDevice>;
+}
+
+```
+
+You should see either "processor is supported" or "processor is not supported" in your browser developer tools based on whether 
+Background Blur is supported on your device and browser version.
+
+You can access the state by using the [useBackgroundBlur](/docs/sdk-hooks-usebackgroundblur--page) hook.
+
+## Props
+
+<Props of={BackgroundBlurProvider} />
+
+## Importing
+
+```javascript
+import { BackgroundBlurProvider } from 'amazon-chime-sdk-component-library-react';
+```
+
+## Usage
+
+```jsx
+import React from 'react';
+import { VideoTransformDevice } from 'amazon-chime-sdk-js';
+import {
+  MeetingProvider,
+  BackgroundBlurProvider,
+  useMeetingManager,
+  useBackgroundBlur,
+} from 'amazon-chime-sdk-component-library-react';
+
+const App = () => (
+  <BackgroundBlurProvider>
+    <MeetingProvider>
+      <MyChild />
+    </MeetingProvider>
+  </BackgroundBlurProvider>
+);
+
+const MyChild = () => {
+  const meetingManager = useMeetingManager();
+  const [isVideoTransformCheckBoxOn, setisVideoTransformCheckBoxOn] = useState(false);
+  const [device, setDevice] = useState(meetingManager.selectedVideoInputDevice);
+  const { isBackgroundBlurSupported, createBackgroundBlurDevice } = useBackgroundBlur();
+
+  useEffect(() => {
+    meetingManager.subscribeToSelectedVideoInputTransformDevice(setDevice);
+    return () => {
+      meetingManager.unsubscribeToSelectedVideoInputTransformDevice(setDevice);
+    };
+  }, []);
+
+  useEffect(() => {
+    async function toggleBackgroundBlur() {
+      let current = device;
+      if (isVideoTransformCheckBoxOn) {
+        current = await createBackgroundBlurDevice(device);
+      } else {
+        if (isVideoTransformDevice(device)) {
+          current = await device.intrinsicDevice();
+        }
+      }
+      await meetingManager.selectVideoInputDevice(current);
+    }
+    
+    toggleBackgroundBlur();
+  }, [isVideoTransformCheckBoxOn]);
+
+  const onClick = () => {
+    setisVideoTransformCheckBoxOn(current => !current);
+  };
+
+  return (
+    <div>
+      {isBackgroundBlurSupported && (
+        <button onClick={onClick}>
+          {isVideoTranformDevice(device) ? 'Background Blur Enabled' : 'Enable Background Blur'}
```

Comment:
We would need to add `isVideoTranformDevice(device)` implementation to this example.